From 6e30fc90afe0245ab61bf28e063c06c7e90b20c1 Mon Sep 17 00:00:00 2001
From: Laurentiu Tudor <laurentiu.tudor@nxp.com>
Date: Tue, 27 Jun 2017 17:41:32 +0300
Subject: [PATCH 371/448] staging: fsl-mc: fix a few implicit includes

Few files using byte order macros but did not explicitly
included the required kernel header, so add it.

Signed-off-by: Laurentiu Tudor <laurentiu.tudor@nxp.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/staging/fsl-mc/bus/dpio/dpio.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/staging/fsl-mc/bus/dpio/dpio.c b/drivers/staging/fsl-mc/bus/dpio/dpio.c
index eef58c9..521bc69 100644
--- a/drivers/staging/fsl-mc/bus/dpio/dpio.c
+++ b/drivers/staging/fsl-mc/bus/dpio/dpio.c
@@ -4,6 +4,7 @@
  * Copyright 2016 NXP
  *
  */
+#include <linux/kernel.h>
 #include <linux/fsl/mc.h>
 
 #include "dpio.h"
-- 
1.7.1

